terragrunt/aws/iam/oidc.tf

```diff
@@ -0,0 +1,18 @@
+module "github_oidc_role" {
+  source = "github.com/cds-snc/terraform-modules//gh_oidc_role?ref=v4.0.0"
```

Comment:
I am not sure why you need an oidc_role here - it is usually used for CI/CD which is already setup for you. Can you help me understand why you need this oidc role?

Reply:
Yes, this is for the github action that builds the frontend and uploads it to s3. I realized after creating it that I could probably use the same role as we use for `terraform apply`. That's the one you're referring to, right?